Automattic/newspack-block-theme
# feat: add new "Paul" style

### All Submissions:






### Changes proposed in this Pull Request:



First pass at a new style for the theme. Named after Paul Reuter.

It uses an off-white background with Jost for the headings and Newsreader for the body.

![screenshot](https://github.com/Automattic/newspack-block-theme/assets/177929/9c5a5642-e076-4bcc-925f-89bc4bd1dcad)

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Edit the site and check the global styles
3. Apply new style
4. Check front-end if it's applied correctly

### Other information:
